maven_iuvs/echelle.py

```diff
@@ -2378,9 +2481,9 @@ def line_fit_initial_guess(wavelengths, spectrum, H_a=20, H_b=170, D_a=80, D_b=1
 
     # Total flux of H and D initial guess: get by integrating around the line. Note that the H bounds as defined
```

Comment:
maybe note that this total is in DN, not kR?

Reply:
The variable names communicate this but sure, it can be in the comment. Fixed by 27e272c0b844ce4141a300613e0396440848fac8